newIDE/app/src/MainFrame/EditorContainers/HomePage/TeamSection/index.js

```diff
@@ -0,0 +1,361 @@
+// @flow
+import * as React from 'react';
+import { Trans } from '@lingui/macro';
+
+import List from '@material-ui/core/List';
+import { Line, Column } from '../../../../UI/Grid';
+
+import {
+  type FileMetadataAndStorageProviderName,
+  type StorageProvider,
+} from '../../../../ProjectsStorage';
+import SectionContainer, { SectionRow } from '../SectionContainer';
+import CircularProgress from '../../../../UI/CircularProgress';
+import useForceUpdate from '../../../../Utils/UseForceUpdate';
+import {
+  type TeamGroup,
+  type TeamMembership,
+  type User,
+} from '../../../../Utils/GDevelopServices/User';
+import { type CloudProjectWithUserAccessInfo } from '../../../../Utils/GDevelopServices/Project';
+import TeamContext from '../../../../Profile/Team/TeamContext';
+import TeamGroupNameField from './TeamGroupNameField';
+import NewTeamGroupNameField from './NewTeamGroupNameField';
+import TeamMemberRow from './TeamMemberRow';
+import { makeDropTarget } from '../../../../UI/DragAndDrop/DropTarget';
+import GDevelopThemeContext from '../../../../UI/Theme/GDevelopThemeContext';
+import {
+  ProjectFileListItem,
+  transformCloudProjectsIntoFileMetadataWithStorageProviderName,
+} from '../BuildSection';
+import { useResponsiveWindowWidth } from '../../../../UI/Reponsive/ResponsiveWindowMeasurer';
+import EmptyMessage from '../../../../UI/EmptyMessage';
+import AlertMessage from '../../../../UI/AlertMessage';
+import Text from '../../../../UI/Text';
+import FlatButton from '../../../../UI/FlatButton';
+import Add from '../../../../UI/CustomSvgIcons/Add';
+
+const sortMembersByNameOrEmail = (a: User, b: User) => {
+  return (a.username || a.email).localeCompare(b.username || b.email);
+};
+
+const DropTarget = makeDropTarget('team-groups');
+
+type GroupWithMembers = {| group: TeamGroup, members: User[] |};
+
+type Props = {|
+  project: ?gdProject,
+  onOpenRecentFile: (file: FileMetadataAndStorageProviderName) => void,
+  storageProviders: Array<StorageProvider>,
+|};
+
+export type TeamSectionInterface = {|
+  forceUpdate: () => void,
+|};
+
+const groupMembersByGroupId = ({
```

Comment:
this function could deserve a few unit tests (it looks complex from the outside, so i'd be scared touching it)

Reply:
You're right, that's a good idea